docs/install.md

```diff
@@ -1,122 +1,121 @@
 
-Before any of the other systems can be set up you will need to start the authenication server. By
-default, PORI authenticates against an instance of [KeyCloak](https://www.keycloak.org/). For convenience we have provided a
-docker container with a default configuration of keycloak. This is the authentication server used
-by our [demo instance](https://pori-demo.bcgsc.ca/). If your institution already has a keycloak
-server then we have more [detailed instuctions on setting up through the GUI](https://github.com/bcgsc/pori/blob/master/docs/auth.md)
-in this repository.
+If your institution regularly works on PORI related projects then we recommend setting up a development instance of the PORI platform which your developers can point their applications to. If you do not have access to something like this, then the easiest way to get the dependencies for whatever part of the PORI platform you are working on up and running is by running the development version of the docker compose configuration found in this repository: [docker-compose.dev.yml](https://github.com/bcgsc/pori/blob/master/docker-compose.dev.yml).
 
-```bash
-docker run \
-    -e KEYCLOAK_USER=admin \
-    -e KEYCLOAK_PASSWORD=<PASSWORD> \
-    -e KEYCLOAK_FRONTEND_URL=<URL> \
-    -p 8443:8334 \
-    -p 8888:8080 \
-    -d \
-    --mount type=bind,source=/etc/ssl/certs/current,target=/etc/x509/https,readonly \
-    bcgsc/pori-auth:latest
+```yaml title="docker-compose.dev.yml"
+--8<-- "./docker-compose.dev.yml"
 ```
 
-For the demo server (excluding password) this looked like
+The demo uses a default keycloak setup with a realm "PORI" and two clients: "GraphKB" and "IPR".
+For convenience there are also a number of default users which all have the default password of "secret".
+
+| Name             | Default in DB | Purpose                                                                                                                                              |
+| ---------------- | ------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| graphkb_importer | GraphKB       | This is the default user that is created when the new GraphKB DB is created. It is an admin user that can be used to add new users or import content |
+| ipr_graphkb_link | GraphKB       | This is the user used by IPR to pull data from GraphKB                                                                                               |
+| iprdemo          | IPR           | This is an admin user in the IPR demo db                                                                                                             |
+| graphkb_admin    | GraphKB       | Admin user for managing content/users in the GraphKB web interface                                                                                   |
+
+
+![default users](../images/pori-keycloak-default-users.png)
```

Comment:
images folder inside docs folder. Fix broken link with this: ./images/pori-keycloak-default-users.png